app/models/api/transliterate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+// Check that the class exists before trying to use it
+if (! class_exists('Transliterator')) {
+    $request->error = 'Class Transliterator not present';
+    $json = $request->invalidAPICall();
+
+    return;
+}
+
+switch ($request->parameters[2]) {
+    case 'serbian':
+        $transliterated_locale = 'Serbian-Latin/BGN';
+        break;
+
+    default:
+        break;
+}
+
+if (! isset($transliterated_locale)) {
+    $request->error = 'Wrong language';
+    $json = $request->invalidAPICall();
+
+    return;
```

Comment:
since we don't support any fallback value to sr-Cyrl, that whole block should be returned directly in the switch case about in the default case.